vue-components/stories/Lookup.stories.ts

```diff
@@ -1,7 +1,7 @@
 import Lookup from '@/components/Lookup';
 import Icon from '@/components/Icon';
-import { Component } from 'vue';
 import { MenuItem } from '@/components/MenuItem';
+import { Component } from 'vue';
```

Comment:
I guess it makes a lot more sense to keep all the `@/components` imports together, yeah :+1: